pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
Yes, I'm using the Maven plugin in Eclipse and use that to generate the project.  I'll try it the way you suggest and see if that helps.